src/vscode-bicep/package-lock.json

```diff
@@ -80,6 +80,45 @@
         "vscode": "^1.82.0"
       }
     },
+    "../vscode-bicep-ui/packages/components": {
```

Comment:
@majastrz I missed this in your last PR. I think the TPN workflow needs to run `npm ci --install-links` instead. Before I move the deploy pane app to `vscode-bicep-ui` we'd have to use `--install-links` due to an annoying duplicate React instances issue.

Reply:
Got it. I'll send a different PR and close this one